app/models/cedar-metadata-template.ts

```diff
@@ -0,0 +1,22 @@
+import { attr } from '@ember-data/model';
+import OsfModel from './osf-model';
+
+
+/**
+ * CedarMetadataTemplateModel
+ *
+ * @description The cedar metadata template model
+ */
+export default class CedarMetadataTemplateModel extends OsfModel {
+    @attr('fixstring') schemaName!: string;
+    @attr('fixstring') cedarId!: string;
+    @attr('boolean') active!: boolean;
+    @attr('fixstring') template!: string;
```

Comment:
Same here. We don't need the `fixstring` transform.

Reply:
Is this going to be some json-ld object? If so, should it use the `object` transform?